features/step_definitions/stepdefs.js

```diff
@@ -4,7 +4,7 @@ const { Given, When, Then } = require("cucumber");
 
 const sendRequest = async body => {
   const res = await fetch(
-    "http://localhost:4000/api/registration/createNewRegistration",
+    "https://dev-register-a-food-business-service-double.azurewebsites.net//api/registration/createNewRegistration",
```

Comment:
Do we want URLs in here? E.g. if we run the tests locally? Or do we never do that?

Reply:
If we run locally we can change the URL here manually - the point here is to deploy the app to dev then have travis run tests against that URL.